src/main/resources/coursereport.properties

```diff
@@ -1,5 +1,6 @@
 get.error=Request has has some errors, try again
```

Comment:
In addition to punctuation, the "has has" phrase is odd.  This message should be something like, "There have been some errors processing your request.  Please try again later."